validator/sawtooth_validator/journal/publisher.py

```diff
@@ -53,8 +54,11 @@ def __init__(self,
         self._squash_handler = squash_handler
 
     def _build_block(self, chain_head):
-        """ Build a candidate block
+        """ Build a candidate block and construct the consensus object to
+        validate it.
```

Comment:
As we're working on deepening our docstrings, would be nice to see params (name, type and short description), the return value (type and short description), and any "raises"